src/components/icons/BackInWhite.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function BackInWhite({ className }) {
```

Comment:
since you're not using `className` anyway, you really don't need this to be a React component